views/js/controller/Review/review.js

```diff
@@ -13,7 +13,7 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
- * Copyright (c) 2015-2019 (original work) Open Assessment Technologies SA ;
+ * Copyright (c) 2015-22 (original work) Open Assessment Technologies SA ;
```

Comment:
Nitpicking:
```suggestion
 * Copyright (c) 2015-2022 (original work) Open Assessment Technologies SA ;
```

Reply:
Not sure where I picked up this habit...